src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -922,7 +922,7 @@ bool emberAfOperationalCredentialsClusterCSRRequestCallback(app::CommandHandler
 
         response.NOCSRElements        = nocsrElementsSpan;
         response.attestationSignature = signatureSpan;
-        SuccessOrExit(err = commandObj->AddResponseData(commandPath, response));
+        SuccessOrExit(err = commandObj->AddResponse(commandPath, response));
```

Comment:
Again, will emberAfSendImmediateDefaultResponse in the exit block.  Followup OK.